Components/Input/input.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Spark UI</title>
+        <link
+            rel="shortcut icon"
+            href="/Images/spark-ui-favicon.png"
+            type="image/png"
+        />
+        <link rel="stylesheet" href="/Components/Input/input.css" />
+        <link rel="stylesheet" href="/main.css" />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
+        />
+    </head>
+    <body>
+        <!-- header same as landing page -->
+        <nav aria-label="Top bar" class="landing-header">
+            <a href="/index.html" class="landing-header-link">
+                <img
+                    src="/Images/spark-ui-logo.png"
+                    alt="Website Logo"
+                    class="landing-header-logo"
+                />
+                <span class="landing-header-title">Spark UI</span>
+            </a>
+            <a
+                href="https://mohitdhatrak.bio.link/"
+                target="_blank"
+                class="landing-header-link"
+            >
+                <span>Let's Connect</span>
+                <i class="material-icons landing-svg-icon">public</i>
+            </a>
+        </nav>
+
+        <div class="docs-body-container">
+            <div class="side-nav-container">
+                <aside aria-label="Mail menu" class="drawer drawer-scroll">
+                    <h3 class="nested-drawer-title">Getting Started</h3>
+                    <div class="drawer-links-container nested-drawer">
```

Comment:
what do you mean by drawer-links-container?

Reply:
The drawer element we made in css assignment right, i used it here directly as side navigation, that's why drawer. Basically all pages in drawer are links to pages so drawer-links-container